source/client/factories_impl.h

```diff
@@ -31,7 +31,7 @@ class BenchmarkClientFactoryImpl : public OptionBasedFactoryImpl, public Benchma
                             Envoy::Stats::Scope& scope,
                             Envoy::Upstream::ClusterManagerPtr& cluster_manager,
                             Envoy::Tracing::HttpTracerSharedPtr& http_tracer,
-                            absl::string_view cluster_name,
+                            absl::string_view cluster_name, const int sink_stat_prefix,
```

Comment:
Comment about unnecessary const qualifier next to an int argument applies here as well.

Reply:
Done.